app/views/tags/show.html.erb

```diff
@@ -92,17 +92,29 @@
 <% post_count = @posts.count %>
 <div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
   <div>
-    <%= short_number_to_human post_count, precision: 1, significant: false %>
-    <%= 'post'.pluralize(post_count) %>
+    <span title="<%= post_count %> posts with this tag">
+      <%= short_number_to_human post_count, precision: 1, significant: false %>
+      <%= 'post'.pluralize(post_count) %> &nbsp;
+    </span>
+
+    <span title="RSS feed for this tag">
+      <%= link_to tag_path(id: @category.id, tag_id: @tag.id, format: 'rss') do %>
+      <i class="fas fa-rss"></i> RSS &nbsp;
```

Comment:
Whitespace should be moved out of the link, otherwise we end up with this on hover:

![image](https://github.com/user-attachments/assets/0629b5f8-debd-44d6-b8c4-c4fe3d9d6eda)

Reply:
Fixed.